### PR TITLE
Remove empty string and allow for Null values

### PR DESCRIPTION
**Pull request by nour-borgi**

### CODE REVIEW

File: importer/analytics-datastore-clickhouse/clickhouseTables.js [hunk 1]
```diff
@@ -1,119 +1,119 @@
 const queries = [
   // Resource Tables
   `CREATE TABLE organization(
-		facility_id String,
-		id String,
-		version String,
+		facility_id String null,
```
Comment (michaelloosen) [inline importer/analytics-datastore-clickhouse/clickhouseTables.js:4]: Can we please make these capitalized NULL as well to keep it consistent with the rest of the query?

**Review by michaelloosen — CHANGES_REQUESTED** [review]
Looks good, just the one comment

File: importer/analytics-datastore-clickhouse/clickhouseTables.js [hunk 2]
```diff
@@ -1,119 +1,119 @@
...
+		id String null,
+		version String null,
```
Comment (nour-borgi) [inline importer/analytics-datastore-clickhouse/clickhouseTables.js:6]:
```suggestion
		facility_id String NULL,
		id String NULL,
		version String NULL,
```

File: importer/analytics-datastore-clickhouse/clickhouseTables.js (hunk 1 above)
Comment (nour-borgi) [reply importer/analytics-datastore-clickhouse/clickhouseTables.js]: Done! Thanks

**Review by michaelloosen — APPROVED** [review]
LGTM

**Review by Mark-Labuschagne — APPROVED** [review]

**Pull request merged**